libc-test/semver/linux.txt

```diff
@@ -1888,6 +1888,7 @@ PTRACE_EVENT_VFORK_DONE
 PTRACE_GETEVENTMSG
 PTRACE_GETREGSET
 PTRACE_GETSIGINFO
+PTRACE_GETSIGMASK
```

Comment:
This should be added to https://github.com/rust-lang/libc/blob/master/libc-test/semver/linux-gnu.txt.

Reply:
Yes, or rather, I will add `PTRACE_{GET/SET}SIGMASK` to musl and uclibc as well, as the symbols exist on musl and uclibc, too.

Reply:
Uh, they don't generally on uclibc (only exist for tile platform), but exist on musl

Reply:
Feel free to add if you need the items there.